src/pages/wallet/Asset.tsx

```diff
@@ -68,14 +68,7 @@ const Asset = (props: Props) => {
             <h1 className={styles.price}>
               {currency.symbol}{" "}
               {coinPrice ? (
-                <Read
-                  {...props}
-                  amount={walletPrice}
-                  decimals={decimals}
-                  fixed={2}
-                  denom=""
-                  token=""
-                />
+                <Read denom="" amount={walletPrice} fixed={2} />
```

Comment:
735 SWTH Price should be `$3.04`.

On branch:

<img width="405" alt="Screenshot 2023-08-24 at 12 18 11 AM" src="https://github.com/terra-money/station/assets/30275692/7e86da1e-7fc4-4d03-be7e-15d1cc0a13c0">

In production:

<img width="408" alt="prod" src="https://github.com/terra-money/station/assets/30275692/0d9662be-9abe-46c6-841b-d5bfb1ea2588">

Reply:
fixed